### PR TITLE
Adds new endpoint to distribution module at /s3credentialsREADME

### PR DESCRIPTION
**Pull request by flamingbear**
Addresses [CUMULUS-2350: Add a README to the distribution API explaining direct S3 access](https://bugs.earthdata.nasa.gov/browse/CUMULUS-2350)

## Changes

* Adds a new route to the s3-credentials-endpoint with basic instructions on how to use the s3credentials endpoint.
* Updates terraform to add the endpoint to the distribution gateway.
* Updates webpack and package.json to include the static files needed to be served by the endpoint (and the md file that is not actually needed).
* simple test to make sure the environment is being used in the template.

Test by deploying and navigating your browser to: DISTIBUTION_ENDPOINT/s3credentialsREADME 

## PR Checklist

- [X] Update CHANGELOG
- [X] Unit tests
- [X] Adhoc testing
- [n/a] Integration tests

### CODE REVIEW

File: packages/s3-credentials-endpoint/instructions/index.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  <div id="wrapper">
+<h1 id="in-regiondirects3accesstobuckets">In-region Direct S3 Access to Buckets</h1>
+
+<p>Cumulus deployment api can be configured to dispense temporary credentials for same-region, read-only, direct S3 access.</p>
+
+<h2 id="gettemporarycredentials">Get Temporary Credentials</h2>
+
+<p>You can retrieve temporary S3 credentials at the <code>/s3credentials</code> endpoint when authenticated via earthdata login.</p>
+
+<p><a href="${process.env.DISTRIBUTION_ENDPOINT}/s3credentials">${process.env.DISTRIBUTION_ENDPOINT}/s3credentials</a></p>
```
Comment (flamingbear) [inline packages/s3-credentials-endpoint/instructions/index.html:31]: This is the magic line

File: tf-modules/distribution/README.md [hunk 2]
```diff
@@ -16,17 +16,13 @@ Credentials Endpoint with a configuration targeted at Cumulus and NGAP.
 
 ### Optional
```
Comment (flamingbear) [inline tf-modules/distribution/README.md:17]: This is just removing linebreaks from the markdown so it flows correctly.

File: tf-modules/distribution/main.tf [hunk 3]
```diff
@@ -230,15 +230,56 @@ resource "aws_api_gateway_integration" "s3_credentials_redirect" {
...
-    aws_api_gateway_integration.s3_credentials[0]
+    aws_api_gateway_integration.s3_credentials[0],
+    aws_api_gateway_integration.s3_credentials_readme[0]
   ]
 
+  triggers = {
+    redeployment = sha1(join(",", list(
+      jsonencode( aws_api_gateway_integration.s3_credentials_redirect[0] ),
+      jsonencode( aws_api_gateway_integration.s3_credentials[0] ),
+      jsonencode( aws_api_gateway_integration.s3_credentials_readme[0] ),
```
Comment (flamingbear) [inline tf-modules/distribution/main.tf:277]: I'm following the instructions here : https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/api_gateway_deployment#redeployment-triggers

It seems like it forces a redeploy correctly (but there is still timing with teaCache problem)

**Review by jennyhliu — APPROVED** [review]
Nice work!

**Pull request merged**